src/samples/ChildBenefitsClaim/ConfirmationPage.tsx

```diff
@@ -73,69 +74,75 @@ const ConfirmationPage = ({ caseId }) => {
     return <ShutterServicePage />;
   } else if (!loading && isBornAbroadOrAdopted) {
     return (
-      <MainWrapper>
-        <div className='govuk-panel govuk-panel--confirmation govuk-!-margin-bottom-7'>
-          <h1 className='govuk-panel__title'> {t('APPLICATION_RECEIVED')}</h1>
-          <div className='govuk-panel__body govuk-!-font-size-27'>
-            {t('POST_YOUR_SUPPORTING_DOCUMENTS')}
+      <>
+        <LanguageToggle />
```

Comment:
Instead of adding on multiple places, can we add language toggle to app header component.

Reply:
Done